LICENSE-binary

```diff
@@ -261,7 +260,6 @@ net.sf.supercsv:super-csv
 org.apache.arrow:arrow-format
 org.apache.arrow:arrow-memory
 org.apache.arrow:arrow-vector
-org.apache.commons:commons-configuration2
```

Comment:
curious: these only apply to the default Spark profile? so it's fine even though `hadoop-2.7` profile still uses them?

Reply:
This is not used in `hadoop-2.7`. In this PR, all removal is verified like the following. And, the library used by hadoop-2.7 are not included.

**BRANCH-3.1**
```
deps:branch-3.1 $ git grep commons-configuration2
spark-deps-hadoop-3.2-hive-2.3:commons-configuration2/2.1.1//commons-configuration2-2.1.1.jar
```

**MASTER**
```
deps:master $ git grep commons-configuration2
```

Reply:
We have dependency manifest file for all combinations. So, we can check it easily.

Reply:
Got it. Thanks for the explanation!